optionals/sys_sog/subConfig/sys_sog_vehicles/config.cpp

```diff
@@ -0,0 +1,81 @@
+#include "script_component.hpp"
+
+class CfgPatches {
+    class ADDON {
+        addonRootClass = QUOTE(COMPONENT);
+        units[] = {};
+        weapons[] = {};
+        requiredVersion = REQUIRED_VERSION;
+        requiredAddons[] = {"acre_main", "loadorder_f_vietnam"};
```

Comment:
Why not just add this to the main config?

Reply:
I recalled that somebody somewhere complained that the compat required something more than just ACRE addons as they wanted it always loaded, but I guess it does not matter with `skipWhenMissingDependencies`, will move it.

Reply:
I think when skipping is in stable, we'll add it to all compats and put them in the base anyways.

Or maybe not due to size? Something to decide on.